Improve changelog entry (#1138)
OpenTermsArchive/engine: CHANGELOG.md

### Changed

- **Breaking:** Rename `documents` key to `terms` in service declarations for vocabulary consistency; to update your declarations:

  For Unix-like systems:

  ```bash
  find . -name "*.json" -type f -exec sed -i 's/"documents":/"terms":/g' {} +
  ```

  For macOS:

  ```bash
  find . -name "*.json" -type f -exec sed -i '' 's/"documents":/"terms":/g' {} +
  ```

>   ⚠️ Note: When updating the engine on your declarations repositories, the validation of modified declarations will fail once. This is expected and can be safely ignored

Reviewer: ```suggestion
  ⚠️ Note: when updating the engine, CI checks of modified declarations will fail once. This is expected and can be safely ignored
```
